Added links for securing pollers, updating them and monitoring with them in start-with-pollers.md (#3914)
centreon/centreon-documentation: cloud/getting-started/start-with-pollers.md

2. Then you can follow the [installation](../installation/deploy-poller.md) procedure.

> Once the poller is installed, we recommend [securing](../installation/poller-secure.md) it and making sure they remain [updated](../installation/poller-update-upgrade.md). If you have more than one poller, we also recommend having them [monitor each other](../installation/poller-monitor.md) rather than having each poller monitor itself.

Reviewer: ```suggestion
Once the poller is installed, we recommend [securing](../installation/poller-secure.md) it and keeping it [up-to-date](../installation/poller-update-upgrade.md). If you have more than one poller, we also recommend having them [monitor each other](../installation/poller-monitor.md) rather than having each poller monitor itself.
```

## Troubleshooting

> We have a section dedicated to [troubleshooting pollers](../installation/poller-troubleshoot.md) to help you out, other Centreon users also often help other users on our community platform [The Watch](https://thewatch.centreon.com/).

Reviewer: ```suggestion
We have a section dedicated to [troubleshooting pollers](../installation/poller-troubleshoot.md) to help you out. You can also use [our community platform The Watch](https://thewatch.centreon.com/).
```
